adafruit_tmp11X.py

```diff
@@ -113,10 +116,12 @@ def is_valid(cls, value: int) -> bool:
         """Validate that a given value is a member"""
         return value in cls.string
 
-
 class AverageCount(CV):
     """Options for `averaged_measurements`"""
-
+    AVERAGE_1X = None
+    AVERAGE_8X = None
+    AVERAGE_32X = None
+    AVERAGE_64X = None
```

Comment:
These make the library take more RAM, which is a problem on small boards. Don't include these even if it is more convenient for IDE's.
```suggestion
```